### ISSUE TITLE
Accessibility docs

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#160]
We need to provide separate docs on accessibility features inside of AC itself.

Things that should be noted there include:
- example usage case
- supported environment (browser + reader)
- hotkeys
